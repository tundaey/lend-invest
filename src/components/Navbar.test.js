import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import NavBar from './NavBar'

const title = 'Lend-Invest'

test('NavBar renders properly', ()=> {
    const wrapper = shallow(<NavBar title={title}/>)
    const element = wrapper.find('span');
    expect(element.length).toBe(1);
    expect(element.get(0).props.children).toBe(title) 
})

test('Navbar component renders a snapshot properly', ()=>{
    const tree = renderer.create(<NavBar title={title}/>).toJSON();
    expect(tree).toMatchSnapshot(); 
})