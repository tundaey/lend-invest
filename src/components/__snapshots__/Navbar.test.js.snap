// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders a snapshot properly 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <span
        className="navbar-brand"
      >
        Lend-Invest
      </span>
    </div>
  </div>
</nav>
`;
